### ISSUE TITLE
NoSuchMethodError with Play 2.8.8

### ISSUE THREAD

**Issue by fkrull** [kamon-io/sbt-kanela-runner#15]
After upgrading to Play 2.8.8, the app crashes when trying to start in dev mode with a NoSuchMethodError. Production mode still works fine. Play 2.8.7 still works.

Versions:
sbt 1.4.9
Scala 2.13.3
com.typesafe.play.sbt-plugin 2.8.8
io.kamon.sbt-kanela-runner-play-2.8 2.0.10
io.kamon.kamon-bundle 2.1.15

Output:
```
[info] p.c.s.AkkaHttpServer - Enabling HTTP/2 on Akka HTTP server...
[warn] p.c.s.AkkaHttpServer - No HTTPS server bound. Only binding HTTP. Many user agents only support HTTP/2 over HTTPS.
[info] p.c.s.AkkaHttpServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

(Server started, use Enter to stop and go back to the console...)

Uncaught error from thread [play-dev-mode-akka.actor.default-dispatcher-6]: 'play.runsupport.Reloader$CompileResult play.sbt.run.PlayReload$.compile(scala.Function0, scala.Function0, scala.Function0)', shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[play-dev-mode]
java.lang.NoSuchMethodError: 'play.runsupport.Reloader$CompileResult play.sbt.run.PlayReload$.compile(scala.Function0, scala.Function0, scala.Function0)'
	at play.sbt.run.KanelaPlayRun$.$anonfun$playRunTask$4(KanelaPlayRun.scala:42)
	at kamon.instrumentation.sbt.play.KanelaReloader.$anonfun$reload$1(KanelaReloader.scala:474)
	at kamon.instrumentation.sbt.play.KanelaReloader$$anon$1.run(KanelaReloader.scala:44)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at kamon.instrumentation.sbt.play.KanelaReloader$.kamon$instrumentation$sbt$play$KanelaReloader$$withReloaderContextClassLoader(KanelaReloader.scala:49)
	at kamon.instrumentation.sbt.play.KanelaReloader.reload(KanelaReloader.scala:474)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:147)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at kamon.instrumentation.akka.http.Http2BlueprintInterceptor$HandlerWithEndpoint.apply(AkkaHttpServerInstrumentation.scala:354)
	at kamon.instrumentation.akka.http.Http2BlueprintInterceptor$HandlerWithEndpoint.apply(AkkaHttpServerInstrumentation.scala:351)
	at akka.http.impl.engine.http2.Http2Ext.$anonfun$handleUpgradeRequests$1(Http2.scala:167)
	at akka.stream.impl.fusing.MapAsyncUnordered$$anon$31.onPush(Ops.scala:1418)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.ActorGraphInterpreter$SimpleBoundaryEvent.execute(ActorGraphInterpreter.scala:56)
	at akka.stream.impl.fusing.ActorGraphInterpreter$SimpleBoundaryEvent.execute$(ActorGraphInterpreter.scala:52)
	at akka.stream.impl.fusing.ActorGraphInterpreter$BatchingActorInputBoundary$OnNext.execute(ActorGraphInterpreter.scala:95)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:775)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:790)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:579)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at kamon.instrumentation.executor.ExecutorInstrumentation$InstrumentedForkJoinPool$TimingRunnable.run(ExecutorInstrumentation.scala:678)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
[error] a.a.ActorSystemImpl - Uncaught error from thread [play-dev-mode-akka.actor.default-dispatcher-6]: 'play.runsupport.Reloader$CompileResult play.sbt.run.PlayReload$.compile(scala.Function0, scala.Function0, scala.Function0)', shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[play-dev-mode]
java.lang.NoSuchMethodError: 'play.runsupport.Reloader$CompileResult play.sbt.run.PlayReload$.compile(scala.Function0, scala.Function0, scala.Function0)'
	at play.sbt.run.KanelaPlayRun$.$anonfun$playRunTask$4(KanelaPlayRun.scala:42)
	at kamon.instrumentation.sbt.play.KanelaReloader.$anonfun$reload$1(KanelaReloader.scala:474)
	at kamon.instrumentation.sbt.play.KanelaReloader$$anon$1.run(KanelaReloader.scala:44)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at kamon.instrumentation.sbt.play.KanelaReloader$.kamon$instrumentation$sbt$play$KanelaReloader$$withReloaderContextClassLoader(KanelaReloader.scala:49)
	at kamon.instrumentation.sbt.play.KanelaReloader.reload(KanelaReloader.scala:474)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:147)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at kamon.instrumentation.akka.http.Http2BlueprintInterceptor$HandlerWithEndpoint.apply(AkkaHttpServerInstrumentation.scala:354)
[info] p.c.s.AkkaHttpServer - Stopping Akka HTTP server...
[info] p.c.s.AkkaHttpServer - Terminating server binding for /0:0:0:0:0:0:0:0:9000
[info] p.c.s.AkkaHttpServer - Running provided shutdown stop hooks
```

**Comment by kimnilsson** [comment]
Hey, I encountered the same problem and raised a [PR ](https://github.com/kamon-io/sbt-kanela-runner/pull/14)that seems to fix this.

**Comment by antoine-morisseau** [comment]
I have the same issue.
Hope to see the PR of @kimnilsson merged and released soon.

**Comment by alejandrod-f** [comment]
Same issue! will be good to make sure it works with SBT 1.5.3

**Comment by sblausten** [comment]
Same issue here also

**Comment by RichardSmithONS** [comment]
I've just encountered this error, I guess the aforementioned PR is still waiting to be merged. 

I'm using:
sbt 1.5.5
Scala 2.13.6
com.typesafe.play.sbt-plugin 2.8.8
io.kamon.sbt-kanela-runner-play-2.8 2.0.10.15
io.kamon.kamon-bundle 2.2.3

**Comment by alejandrod-f** [comment]
FYI I do not have this issue anymore. I upgraded kamon an all good. I also force the kanela version to be the latest. `1.0.11`

**Comment by raushankumar27** [comment]
Is there any workaround for this issue?
I tried with "io.kamon" % "kanela-agent" % "1.0.11"  but the issue is still present

**Comment by gwak** [comment]
I have the same issue, I had to revert to Play 2.8.7.
Referenced here https://github.com/kamon-io/Kamon/discussions/1056

**Comment by ivantopo** [comment]
Folks, this is solved now on version `2.0.12`. There were binary incompatible changes in an internal method in Play's SBT plugin that messed with some nasty things we are doing to enable Kamon in development mode. 

It took forever to publish it because we had some issues with publishing to Sonatype after Bintray was sunset, but we got over those issues now. Upgrade to `2.0.12` for Play 2.8.8 and let us know if you have any issues.

**Comment by RichardSmithONS** [comment]
Worked for me.
